projects/vdk-plugins/vdk-csv/src/vdk/plugin/csv/csv_plugin.py

```diff
@@ -83,6 +93,65 @@ def ingest_csv(ctx: Context, file: str, table_name: str, options: str) -> None:
     click.echo(f"Ingesting csv file {file} finished.")
 
 
+@hookimpl
+def vdk_configure(config_builder: ConfigurationBuilder) -> None:
+    """
+    Here we define what configuration settings are needed for sqlite with reasonable defaults
+    """
+    sqlite_configuration.add_definitions(config_builder)
+
+
+@hookimpl
+def initialize_job(context: JobContext) -> None:
+    conf = SQLiteConfiguration(context.core_context.configuration)
+    context.connections.add_open_connection_factory_method(
+        "SQLITE",
+        lambda: SQLiteConnection(pathlib.Path(conf.get_sqlite_file())).new_connection(),
+    )
+
+    context.ingester.add_ingester_factory_method(
+        "sqlite", (lambda: IngestToSQLite(conf))
+    )
+
+
+@click.command(
+    name="export-csv",
+    help="Execute a SQL query against a local SQLite database and export to a local CSV file.",
+)
+@click.option("-q", "--query", type=click.STRING, required=True)
+@click.option("-n", "--name", type=click.STRING, default="result.csv", required=False)
+@click.option("-p", "--path", type=click.STRING, default="", required=False)
+@click.pass_context
+def export_csv(ctx: click.Context, query, name: str, path: str):
+    conf = SQLiteConfiguration(ctx.obj.configuration)
+    conn = SQLiteConnection(conf.get_sqlite_file())
+
+    with closing_noexcept_on_close(conn.new_connection().cursor()) as cursor:
```

Comment:
Similar to the previous comment, we'd like this command to work with any configured database. The conf and conn variables should be dropped, and you'd need to get the database cursor through an alternative way; I believe it's possible to do it through the click Context object, but @tozka might be able to clarify.

Reply:
Gabi is correct. It needs to work for any configured database. 

The idea is to do the same thing as in ingest-csv. Copy paste the code and change it to export (instead of ingest) . ingest-csv starts a "data job" in order to ingest the data. So we need to create new data job dir that exports (instead of ingest) data and call it from `export-csv` command.

This way user can configure  DB_DEFAULT_TYPE to decide to what database they want to export. 
You can see doc of  db_default_type using `vdk config-help`